pyof/foundation/basic_types.py

```diff
@@ -237,18 +242,20 @@ class IPAddress(GenericType):
     netmask = UBInt32()
     max_prefix = UBInt32(32)
 
-    def __init__(self, address="0.0.0.0/32"):
+    def __init__(self, value=None):
```

Comment:
Thinking about the user, it might be easier to see default values in current code. In Python, only mutable objects require using None in some cases. Do you have other argument than making all the code look the same?